c2corg_ui/templates/whatsnew.html

```diff
@@ -0,0 +1,42 @@
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
+
+<%block name="metarobots">
+  <meta name="robots" content="index,follow">
+</%block>
+
+<%block name="moduleConstantsValues">
+  ## Set in the data request
+  module.value('mapFeatureCollection', null);
+
+</%block>
```

Comment:
That's already the default value:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/base.html#L153-L155
You don't need to override it.